### PR TITLE
5.10 additional fixes and additions

### PR DESCRIPTION
**Pull request by jeffreytoffoli**
Fixed image links that were not working, added a bit more content to support 5.10 release, fixed a small issue in curl that was causing issues for customers

### CODE REVIEW

File: api-guide/predict/images.md [hunk 1]
```diff
@@ -438,14 +438,14 @@ curl -X POST \
       }
     ]
   }'\
-  https://api.clarifai.com/v2/models/@@generalModelId/outputs
+  https://api.clarifai.com/v2/models/generalModelId/outputs
```
Comment (zeiler) [inline api-guide/predict/images.md:441]: do we make it clear somewhere that `generalModelId is not the actual ID of the general model? 

Usually we should do `{GENERAL_MODEL_ID}` to make that more clear it's something you need to fill in as a user.

Comment (zeiler) [reply api-guide/predict/images.md:441]: there are probably lots of places in the docs that need to be improved like this.

Comment (jeffreytoffoli) [reply api-guide/predict/images.md:441]: on it

Comment (jeffreytoffoli) [reply api-guide/predict/images.md]: the docs do mention that you specify the model you'd like to use with the `{model-id}` parameter, and this convention is used throughout the code samples, so I will use this. a systematic review of all of the samples is something that is on my to-do list

File: api-guide/predict/images.md [hunk 2]
```diff
@@ -2,7 +2,7 @@
 
 ## Via URL
 
-To get predictions for an input, you need to supply an image and the model you'd like to get predictions from. You can supply an image either with a publicly accessible URL or by directly sending bytes. You can send up to 128 images in one API call. You specify the model you'd like to use with the `{model-id}` parameter.
+To get predictions for an input, you need to supply an image and the model you'd like to get predictions from. You can supply an image either with a publicly accessible URL or by directly sending bytes. You can send up to 128 images in one API call. genera
```
Comment (zeiler) [inline api-guide/predict/images.md:5]: not sure what happened in this last sentence.

**Pull request merged**